.rubocop_todo.yml

```diff
@@ -1,18 +1,103 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-08 14:05:42 -0700 using RuboCop version 0.34.2.
+# on 2023-03-17 10:29:23 UTC using RuboCop version 1.28.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 1
+# This cop supports safe auto-correction (--auto-correct).
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/DateAssignment:
```

Comment:
I'd be tempted to just fix this by dropping the `.date = ...` bit in gemspec.

Reply:
done